polars/polars-io/src/parquet/read.rs

```diff
@@ -6,9 +6,31 @@ use crate::prelude::*;
 use crate::RowCount;
 use arrow::io::parquet::read;
 use polars_core::prelude::*;
+#[cfg(feature = "serde")]
+use serde::{Deserialize, Serialize};
 use std::io::{Read, Seek};
 use std::sync::Arc;
 
+#[derive(Copy, Clone, Debug)]
+#[cfg_attr(feature = "serde", derive(Serialize, Deserialize))]
+pub enum ParallelStrategy {
+    /// Don't parallelize
+    None,
+    /// Parallelize over the row groups
```

Comment:
Is the doc a typo?

Reply:
Why?

Reply:
seems `Parallelize over the row groups` should be the comment of `RowGroups`

Reply:
Ah, now I see it. Jup, that's wrong.